File: 18-Functions-Exercises/49-list_manipulation.js
function listManipulation(arr, cmd, loc, val) {
    if (cmd === 'remove') {
        if (loc === 'end') return arr.pop()
        if (loc === 'beginning') return arr.shift()
    }
    if(cmd === 'add') {
        if (loc === 'beginning') {
            arr.unshift(val)
            return arr
        }
        if (loc === 'end') {
            arr.push(val)
            return arr
        }
    }
}

File: 18-Functions-Exercises/49-list_manipulation.py
def list_manipulation(lst, cmd, loc, val=None):
    if cmd == 'remove':
        if loc == 'end':
            return lst.pop()
        elif loc == 'beginning':
            return lst.pop(0)
    elif cmd == 'add':
        if loc == 'beginning':
            lst.insert(0, val)
            return lst
        elif loc == 'end':
            lst.append(val)
            return lst


